packages/react-native-fbsdk/README.md

```diff
@@ -0,0 +1,50 @@
+# @config-plugins/react-native-fbsdk
```

Comment:
Wanna name this `react-native-fbsdk-next` or is the plan to eventually change the upstream package name?

Reply:
yeah, the name should be `@config-plugins/react-native-fbsdk-next`. This PR just is a working space for testing the plugin. I will copy it to the upstream package.

Reply:
Great, one other note, I think we probably don't want to merge any native code under the `@config-plugins` namespace since it's more of a set of workaround packages. It'd be like if `@types/node` had some extra JS code that you'd depend on to get Node.js working.

Reply:
If upstream package dont accept the plugin, so where we can put the native (adapter) code?